src/app/projects/add-edit-project/add-edit-project.component.ts

```diff
@@ -82,6 +82,12 @@ export class AddEditProjectComponent implements OnInit, AfterViewInit, OnDestroy
     'Plan Implementation and Monitoring'
   ];
 
+  public projectTypes: Array<Object> = [
```

Comment:
You can use the ProjectTypes type that you defined in project.ts here

Reply:
done